### ISSUE TITLE
some types and function not exported

### ISSUE THREAD

**Issue by vinayr** [dabakovich/react-native-controlled-mentions#42]
In readme `Rendering MentionInput's value`

```
import {
  Part,
  PartType,
  parseValue,
  isMentionPartType,
} from 'react-native-controlled-mentions';
```
`Part`, `PartType`, `isMentionPartType` are not exported.

**Comment by dabakovich** [comment]
Thanks for the reported bug!
Yeah, I see. Will publish a patch soon.

**Comment by dabakovich** [comment]
Just published the v2.2.5 patch update with the fix.

**Comment by vinayr** [comment]
Thank you
